includes/Admin/Settings.php

```diff
@@ -477,6 +477,22 @@ public function get_settings_fields() {
                     'sanitize_callback'          => 'wc_format_decimal',
                     'response_sanitize_callback' => 'wc_format_localized_price',
                 ],
+                'withdraw_order_status' => [
+                    'name'    => 'withdraw_order_status',
+                    'label'   => __( 'Order Status for Withdraw', 'dokan-lite' ),
+                    'desc'    => __( 'Order status for which vendor can make a withdraw request.', 'dokan-lite' ),
+                    'type'    => 'multicheck',
+                    'default' => array(
```

Comment:
default option should be like this:
array( 'wc-completed' => 'wc-completed', 'wc-processing' => 'wc-processing' ),
also remove wc-on-hold from default option.
don't remove wc-on-hold from options